frontend/src/scenes/data-management/database/DatabaseScene.tsx

```diff
@@ -0,0 +1,108 @@
+import { PageHeader } from 'lib/components/PageHeader'
+import { DataManagementPageTabs, DataManagementTab } from 'scenes/data-management/DataManagementPageTabs'
+import { SceneExport } from 'scenes/sceneTypes'
+import { databaseSceneLogic } from './databaseSceneLogic'
+import { useValues } from 'kea'
+import { LemonTable } from 'lib/lemon-ui/LemonTable/LemonTable'
+import { LemonTag } from 'lib/lemon-ui/LemonTag/LemonTag'
+import { Link } from '@posthog/lemon-ui'
+import { urls } from 'scenes/urls'
+
+export function DatabaseScene(): JSX.Element {
+    const { database } = useValues(databaseSceneLogic)
+    const tables = database ? Object.keys(database) : []
+
+    return (
+        <div data-attr="database-scene">
+            <PageHeader
+                title="Data Management"
+                caption="Use data management to organize events that come into PostHog. Reduce noise, clarify usage, and help collaborators get the most value from your data."
+                tabbedPage
+            />
+            <DataManagementPageTabs tab={DataManagementTab.Database} />
+
+            <div className="mb-4">
+                These are the database tables you can query in PostHog through{' '}
+                <a href="https://posthog.com/manual/hogql" target="_blank">
+                    HogQL
+                </a>
+                .
+            </div>
+
+            {tables.length === 0 ? <LemonTable loading={true} dataSource={[]} columns={[]} /> : null}
+
+            {tables.map((table) => (
+                <div key={table} className="mt-8">
+                    <h2 className="subtitle">Table: {table}</h2>
+
+                    <LemonTable
+                        dataSource={database[table]}
+                        columns={[
+                            {
+                                title: 'Column',
+                                key: 'key',
+                                dataIndex: 'key',
+                                render: function RenderColumn(column) {
+                                    return <code>{column}</code>
+                                },
+                            },
+                            {
+                                title: 'Type',
+                                key: 'type',
+                                dataIndex: 'type',
+                                render: function RenderType(type) {
+                                    if (type === 'virtual_table') {
+                                        return <LemonTag type="default">Virtual Table</LemonTag>
```

Comment:
not blocking right now but as someone fresh to this page I'm going to wonder "oh, what's a (virtual table|reference|expression)" I'll have an idea as a technical user but we should probably make a link to a section of the docs obvious here in an info button or something